src/register/register.ts

```diff
@@ -146,10 +316,49 @@ export class Register {
         })
       );
 
-      Register.registers[register].text = text;
-    }
+      // Harmonize newline character
+      text = text.replace(/\r\n/g, '\n');
```

Comment:
Ah, very interesting behaviour. I repro that on my mac and I see what you mean, in that case using `\n` is the way to go.